pom.xml

```diff
@@ -85,7 +85,7 @@
         <slf4j.version>1.7.22</slf4j.version>
         <tiwulfx.version>2.0</tiwulfx.version>
 
-        <powsyblafs.version>3.1.0</powsyblafs.version>
+        <powsyblafs.version>3.2.0-SNAPSHOT</powsyblafs.version>
```

Comment:
Is it a mistake?

Reply:
no

Reply:
Should we make a PR for that ? As mentionned afs and gse are so coupled that I think we should track current snapshot for dev.